### DISCUSSION TITLE
can 't use a dropdown inside dropdown

### DISCUSSION THREAD

**Discussion by alvarolm** [fomantic/Fomantic-UI#2991] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)]
i need to use a searchable dropdown inside another dropdown.
the embedded dropdown it seems to auto hide every time is activated

https://jsfiddle.net/L1nw5zc3/

any guidance is appreciated

**Comment by lubber-de** [comment]
Needs investigation as this use case does not seem to be properly supported.
In the meantime, would it help to use a submenu instead?

https://jsfiddle.net/lubber/s70twju3/1/

![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/db6ad131-e223-4f5c-a873-b8f298ea92a1)

**Reply by alvarolm** [reply]
in my use case the select options are over 200, too many to be displayed at once or scrolling one by one, that is why the searchbox is needed
